src/components/Notify/NotifyForm.tsx

```diff
@@ -1,22 +1,76 @@
+import React, { useContext, useEffect, useState } from 'react';
 import {
   View,
   Text,
   TextInput,
-  Picker,
-  Button,
   StyleSheet,
+  TouchableOpacity,
+  FlatList,
+  Image,
+  Keyboard,
 } from 'react-native';
-import React, { useState } from 'react';
+import Colors from '../../constants/colors/Colors';
+import StyleConfig from '../../utils/StyleConfig';
+import { scale } from '../../utils/utils';
+import {
+  getAllUsers,
+  postFcmToken,
+  sendNotification,
+} from '../../screens/AuthScreen/Util';
+import { AuthContext } from '../../context/AuthContext';
+import { firebase } from '@react-native-firebase/messaging';
 
 const NotifyForm = () => {
   const [title, setTitle] = useState('');
   const [description, setDescription] = useState('');
-  const [notifyTo, setNotifyTo] = useState('');
+  const [isDropDownSelected, setIsDropDownSelected] = useState(false);
+  const [searchQuery, setSearchQuery] = useState('');
+  const [allUsers, setAllUsers] = useState([]);
+  const [selectedUser, setSelectedUser] = useState('');
+  const [isLoading, setIsLoading] = useState(true);
+  const { loggedInUserData } = useContext(AuthContext);
+  const token = loggedInUserData?.token;
+
+  const selectDropDown = () => {
+    Keyboard.dismiss();
+    setIsDropDownSelected(!isDropDownSelected);
+  };
 
-  const handleButtonPress = () => {
+  const handleDropDownPress = (item: any) => {
+    setSelectedUser(item);
+    setIsDropDownSelected(false);
+  };
+
+  const getFCMToken = async () => {
+    const permission = await firebase.messaging().hasPermission();
+    if (permission) {
+      const fcmToken_ = await firebase.messaging().getToken();
+
+      await postFcmToken(fcmToken_, token);
+    } else {
+      await firebase.messaging().requestPermission();
+    }
+  };
+  const handleButtonPress = async () => {
     // Handle the button press and perform necessary actions (e.g., send notification)
-    console.log('Notification sent:', { title, description, notifyTo });
+    console.log('setSelected User', {
+      title,
+      description,
+      notifyTo: selectedUser?.id,
+    });
+    await sendNotification(title, description, selectedUser?.id, token);
   };
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const allUser = await getAllUsers(loggedInUserData?.token);
+      setAllUsers(allUser);
+      setIsLoading(false);
+    };
```

Comment:
- just curious not related to this pr, do we make this API call everytime the component is mounted or we cache the API response on mobile?